### PR TITLE
Added parameter `TendrlContext.cluster_name` for the flow

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/node-agent#602
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core @rishubhjain @r0h4n plz review

This is required for https://github.com/Tendrl/commons/pull/720

**Comment by rishubhjain** [comment]
@shtripat Can you add self.cluster_name to https://github.com/Tendrl/monitoring-integration/blob/master/tendrl/monitoring_integration/objects/trigger/__init__.py#L6

Also I think I missed self.integration_id

File: tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ namespace.monitoring:
       inputs:
         mandatory:
           -  TendrlContext.integration_id
+          -  TendrlContext.cluster_name
```
Comment (r0h4n) [inline tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml]: Add the new input parameter here too https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/monitoring_utils.py#L9

Comment (shtripat) [reply tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml]: Ah yes while testing I had it here. Forgot to post in PR somehow. Will update the PR

**Comment by shtripat** [comment]
>> @shtripat Can you add self.cluster_name to https://github.com/Tendrl/monitoring-integration
>> /blob/master/tendrl/monitoring_integration/objects/trigger/__init__.py#L6

Is its really used down the line. If required we can add

File: tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml (hunk 1 above)
Comment (shtripat) [reply tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml]: Added as part of https://github.com/Tendrl/commons/pull/720

**Pull request merged**